src/workerd/io/trace-stream.c++

```diff
@@ -972,20 +972,11 @@ kj::Promise<WorkerInterface::CustomEvent::Result> TailStreamCustomEvent::sendRpc
 
   this->capFulfiller->fulfill(kj::mv(cap));
 
-  // Forked promise for completion of all capabilities associated with the cap stream. This is
```

Comment:
@mar-cf We're removing a bunch of comments here and are set to have no documentation for the try/catch block which will make it difficult to understand, can we add some updated comments?

I'm still somewhat concerned that this kind of change will lead to us getting many cancellation warnings on the edge again. If the concern about returning from the customEvent before the capability is cleaned up is moot, can you explain why?